src/odemis/gui/conf/data.py

```diff
@@ -512,7 +450,7 @@
                 "accuracy": 3,
             },
         },
-    "photo-detector0":  # TODO: for every photo-detector* => make it a regex
+    r"photo-detector.*":
```

Comment:
no need for the `$` here?

Reply:
We could make it as specific as the regex in the path, but it's an overkill. For now, every role which starts with "photo-detector" will be treated by the GUI as a photo-detector. I think it's fair, and if we want more fine tuning, we'll need to update the code.